ga/25.0.0.1/full/Dockerfile.ubi.ibmjava8

```diff
@@ -0,0 +1,51 @@
+# (C) Copyright IBM Corporation 2023.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG PARENT_IMAGE=icr.io/appcafe/websphere-liberty:kernel-java8-ibmjava-ubi
```

Comment:
The tags of the base image for all the 'full' images in 25.0.0.1 folder should include the prefix `25.0.0.1-`